frontend/src/utils/dateUtils.js

```diff
@@ -0,0 +1,31 @@
+export const getTimeZoneAbbreviation = (tzName = Intl.DateTimeFormat().resolvedOptions().timeZone) => {
```

Comment:
Please don't make a new folder for no reason. Also, timezone abbreviations isn't really necessary if you know it's always local. It also kind of looks really bad in the UI, so we would just not at all show timezones in that case.

Reply:
We don't have a utils file. 

Just put it in `components/utils.jsx` instead (: